Library/Homebrew/cmd/audit.rb

```diff
@@ -659,6 +659,28 @@ def audit_line(line, lineno)
     end
   end
 
+  def audit_prefix_has_contents
+    ps = formula.opt_prefix
+    a = "#{ps}/bin"
+    b = "#{ps}/lib"
+    c = "#{ps}/libexec"
+    d = "#{ps}/sbin"
+
+    if Dir["#{a}/*"].empty?
+      if Dir["#{b}/*"].empty?
+        if Dir["#{c}/*"].empty?
+          if Dir["#{d}/*"].empty?
+            problem <<-EOS.undent
+              The installation seems to be empty. Please ensure the prefix
+                 is set correctly and expected files are installed.
+                 The prefix configure/make argument may be case-sensitive.
+            EOS
+          end
+        end
+      end
+    end
+  end
```

Comment:
I was probably just being tired/stupid. I'll take another look later today. Thanks for the code snippets by the way, :heart: for the tidier, better solutions.